index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Eze Promise</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <nav>
+          <button class="menu">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </button>
+        </nav>
+      </header>
+      <section class="headline">
+        <div class="intro">
+          <h1>Hey there. I’m Promise <span class="title">I’m a software developer</span></h1>
+          <p class="more-info"> I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+          <div class="socials">
+            <img class="icon" alt="twitter" src="./img/twitter.png">
+            <img class="icon" alt="linkedin" src="./img/linkedin.png">
+            <img class="icon" alt="medium" src="./img/medium.png">
+            <img class="icon" alt="github" src="./img/github.png">
+            <img class="icon" alt="angellist" src="./img/angellist.png">
+          </div>
```

Comment:
[  OPTIONAL ] The list of icons should use a ul tag to group them, and li per icon; each icon has a link (a tag)

Reply:
Thank you @SajjadAhmad14. I will implement your suggestion.